### PR TITLE
Fix ineffassign issues

### PR DESCRIPTION
**Pull request by zhijianli88**
All detected by [goreportcard](https://goreportcard.com/report/github.com/kubernetes/test-infra)

### CODE REVIEW

File: prow/plugins/project/project.go [hunk 1]
```diff
@@ -234,7 +234,7 @@ func handle(gc githubClient, log *logrus.Entry, e *github.GenericCommentEvent, p
 	var projectID int
```
Comment (fejta) [inline prow/plugins/project/project.go:234]: Move project id declaration down to 252?

File: prow/cmd/tackle/main.go [hunk 2]
```diff
@@ -630,7 +630,8 @@ func ingress(kc *kubernetes.Clientset, ns, service string) (url.URL, error) {
 		if hasResource(kc.Discovery(), networking.SchemeGroupVersion.WithResource("ingresses")) {
 			ing, err = kc.NetworkingV1beta1().Ingresses(ns).List(context2.TODO(), metav1.ListOptions{})
 		} else {
-			oldIng, err := kc.ExtensionsV1beta1().Ingresses(ns).List(context2.TODO(), metav1.ListOptions{})
+			var oldIng *v1beta1.IngressList
+			oldIng, err = kc.ExtensionsV1beta1().Ingresses(ns).List(context2.TODO(), metav1.ListOptions{})
 			if err == nil {
 				ing, err = toNewIngress(oldIng)
```
Comment (fejta) [inline prow/cmd/tackle/main.go:636]: tricky

**Review by fejta — APPROVED** [review]
thanks for this!

File: prow/cmd/tackle/main.go [hunk 3]
```diff
@@ -630,7 +630,8 @@ func ingress(kc *kubernetes.Clientset, ns, service string) (url.URL, error) {
 		if hasResource(kc.Discovery(), networking.SchemeGroupVersion.WithResource("ingresses")) {
 			ing, err = kc.NetworkingV1beta1().Ingresses(ns).List(context2.TODO(), metav1.ListOptions{})
 		} else {
-			oldIng, err := kc.ExtensionsV1beta1().Ingresses(ns).List(context2.TODO(), metav1.ListOptions{})
+			var oldIng *v1beta1.IngressList
```
Comment (fejta) [inline prow/cmd/tackle/main.go:633]: Looks like you lost a merge race here (this import was renamed)

File: prow/cmd/tackle/main.go [hunk 4]
```diff
@@ -33,6 +33,7 @@ import (
 
 	"github.com/sirupsen/logrus"
 	corev1 "k8s.io/api/core/v1"
+	v1beta1 "k8s.io/api/extensions/v1beta1"
```
Comment (fejta) [inline prow/cmd/tackle/main.go:36]: Please follow the pattern: extensionsv1beta1

Comment (zhijianli88) [reply prow/cmd/tackle/main.go]: Fixed

**Comment by rmohr** [comment]
@Katharine @fejta, in kubevirt some people lately did some great ineffassign PRs too. Since I found their findings so useful I looked into the ineffassign tool (https://github.com/kubevirt/kubevirt/pull/4909) and it turns out that it can be used in `nogo` right away: https://github.com/kubevirt/kubevirt/pull/4909.

**Review by fejta — APPROVED** [review]

**Pull request merged**